jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
```

Comment:
"`$name`" is a very generic name; it could be the name of anything. I see that when this script is called, this argument comes from [`$env:APPVEYOR_PROJECT_SLUG`](https://www.appveyor.com/docs/environment-variables), which is the "username/repo" string.

So I suggest you name this "`$projectSlug`".